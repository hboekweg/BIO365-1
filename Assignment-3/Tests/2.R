source("2.R")

print(getBabyName("dog")) # puppy
print(getBabyName("cat")) # kitten
print(getBabyName("horse")) # foal
print(getBabyName("pig")) # piglet
print(getBabyName("bear")) # cub
print(getBabyName("alligator")) # hatchling