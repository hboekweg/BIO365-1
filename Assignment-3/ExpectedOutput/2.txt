[1] "puppy"
[1] "kitten"
[1] "foal"
[1] "piglet"
[1] "cub"
[1] "hatchling"
